src/main/java/dk/bec/polonez/reservationsystem/controller/ReservationController.java

```diff
@@ -48,4 +48,19 @@ public boolean deleteReservation(@PathVariable Long id) {
     public ResponseReservationDto updateReservation(@RequestBody UpdateReservationDto reservationDto) {
         return reservationService.updateReservation(reservationDto);
     }
+
+    @GetMapping("mine")
+    public List<ResponseReservationDto> getAllMyReservations() {
```

Comment:
**advice** I think we should have only one (single) endpoint ```GET /reservations``` and then - depending on a user's role checked in runtime - the server would filter the reservations and return only the matching ones. For example if an endpoint was called by PlaceOwner, the endpoint would return only reservations owned by PlaceOwner. And if it's called by system admin, it would return all reservations. Does it make sense? Such approach would be more REST-full.